### PR TITLE
[11.0][ADD] mrp_bom_tracking

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
MRP BoM Tracking
================

This module adds track visibility to some fields of mrp boms.

### CODE REVIEW

File: mrp_bom_tracking/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adds track visibility to some fields of mrp boms.
```
Comment (LoisRForgeFlow) [inline mrp_bom_tracking/readme/DESCRIPTION.rst]: Can you elaborate a bit more?, it is not just that, it log in the chatter any change in the bom lines too.

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
It's good now :+1: Let's go merge

**Comment by bealdav** [comment]
This module works as it in v12.
To be include in readme ? after merge.

**Comment by LoisRForgeFlow** [comment]
@bealdav please, could you check the migration https://github.com/OCA/manufacture/pull/380

**Pull request closed without merging**